CHANGELOG.md

```diff
@@ -1,3 +1,105 @@
+### 2020-09-14_1.197.3
```

Comment:
The version in the yaml looks to be `2020-09-14_1.196.0`. Do we know why that is?